Kajabi/sage-lib
# revert: separation of heading body font weights

## Description
This reverts the font-weight values from *25 back to *00.



## Testing in `sage-lib`
1. Navigate to Foundations -> Typography
2. Inspect the Headers and body to confirm they use 100-based values.


## Related
https://kajabi.atlassian.net/browse/DSS-752
